src/main/datatypes/MQ2AdvLootType.cpp

```diff
@@ -211,7 +211,8 @@ enum class AdvLootItemMembers
 	AlwaysGreed,
 	Never,
 	IconID,
-	NoDrop
+	NoDrop,
+	FG
```

Comment:
what does FG mean in this context?

Reply:
Free Grab (should call it that)

Reply:
Yes lets call it that.

Reply:
Might want to change it on the item as well

Reply:
I was just informed this was direct eqlib data, not an MQ construct, so changing the name upstream is probably out of scope.

Reply:
> I was just informed this was direct eqlib data, not an MQ construct, so changing the name upstream is probably out of scope.

what's best to do here? overload it so FG and FreeGrab exist? or just leave it FG?

Reply:
Just change the datatype member to FreeGrab

Reply:
dannuic was talking about eqlib which is the source data and is out of scope for this pull request. The request to change FreeGrab covers the changes that are already in this pull request.